woocommerce/woocommerce-gutenberg-products-block
# Product Collection - Add keyword control to sidebar settings

This PR adds `Keyword` control to sidebar settings.
<img width="280" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/9d4e5eb9-4b55-4611-a08c-88eb4292e55f">


Fixes #9365 

### Testing

1. Create a new post
2. Add Product Collection block
3. With the `Product Collection` block selected, check the sidebar for the `Keyword` control. Ensure that it is displayed as shown below:
<img width="285" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/1a75258f-9127-46df-a85d-71d1b3939aa6">
<img width="280" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/9d4e5eb9-4b55-4611-a08c-88eb4292e55f">

5. Add a keyword & changes should immediately reflect in the Editor view. 
6. Save and publish the post.
    - Navigate to the live post on the Frontend and verify that the changes you made in the Editor are accurately reflected.
7. Go back to the Editor and confirm that the filters can be reset using the `Reset all` button, as shown below:
<img width="279" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/0f7b71ab-19ec-4259-8a9f-2f37a8819552">


### WooCommerce Visibility




### Changelog

> Product Collection - Add keyword control to sidebar settings
